### DISCUSSION TITLE
从标准开始想 | /posts/scale/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#600] [Announcements]
https://yuanfan.rbind.io/posts/scale/

**Comment by botsp** [comment]
1.<评分>拜读了，观点很有启发，讲的也好。也了解到怎样看大众评分，早就对那些高评分商家满腹狐疑了、理发店尤甚！

2.关于"客观和全面"应该是在<历史是任人打扮的小姑娘>文章下的讨论，俺也围观了。

3.这两天的月亮真是特别明亮，实在是下楼散步的动力源泉。天涯共明月，顺祝新年快乐~

**Reply by earfanfan** [reply]
其实三水（于淼）就是上次我说还想给你安利的博主，现在你也注意到他啦，真好。

我们老家的说法是“十五的月亮十六圆”，所以今晚的月亮才是一个月最大最圆的，哈哈，千里共婵娟，也祝你新年快乐。

**Comment by zocozocozoco** [comment]
说下我关于你们两个讨论的感想，从讨论开始的「世另我」可能路就歪了，所以前面你一直想论证的是为什么「世另我」——自己和于老师如何相似；但这点在此种讨论中是不是没必要也不重要。

如果从标准来讲的话，是不是已经把「世另我」当做了标准，也许可以跳出来看呢。
（纯讨论）

**Reply by earfanfan** [reply]
嗯，有道理，我那个周末刚好连着看完了《功夫熊猫》第二三部，接着看完三水写的评分心念一动就写留言，确实是我的留言内容和三水博客的内容关系不大。在与三水交流的时候，我总觉得我俩在跨服聊天，不过等我倔强地钻完牛角尖以后发现三水的答案居然正好解开了我的困惑。

另外，其实我之前看到你给三水留言写的是于老师的时候，稍微有点好奇于老师这个称呼是咋来的，这个可以不回答哈。话说我以前在我的博客里面提到三水写的是“三水大人”，后来三水说把大人二字去掉。

**Reply by zocozocozoco** [reply]
没记错的话于老师是山东人，济南一般尊称别人都是「老师」，准确的说是「老师儿」，或者「捞石儿」，叫老师方便。

**Reply by yufree** [reply]
我的全名其实是于有理无理虚数欧氏非欧几何数群论热力磁光声量子原子氢氦锂铍硼碳氮氧氟氖钠镁铝硅磷硫氯氩钾钙分子细胞组织器官种属科界纲门植物动物家庭社区城市国家生态圈地球太阳系银河系宇宙水气声渣微生物循环经济气候变化环境法金木水火土临兵斗者皆阵列前行大慈大悲观世纪福音天主东正伊斯兰飞天拉面经史子集都没搞懂所以很水，里面有三个水，简化为淼字。济南确实都是捞石儿，不特指具体职业。名字这东西无所谓，只要不认错人就好。

**Comment by XiangyunHuang** [comment]
其实，你在 @yufree 博客下的留言，我也没看明白。没想到的是你专门为「钻牛角尖」写了一篇文章。我想说明的是我在《评分》下的留言不是你所谓的吐槽，我只是将我的观点表达出来，不一样没关系，大家互相交流呗。

**Reply by earfanfan** [reply]
好的，俺记住了，我会在自己的脑子里给湘云你专门做个记录，名曰湘云专属忌讳用词名单，已经录入的有喜欢、八卦、吐槽（吐糟）等。

**Reply by XiangyunHuang** [reply]
随了你了。

**Reply by earfanfan** [reply]
嗯，我的脑子里想撒当然随我喽。现在我又想多解释一下了，本来就不是“不一样也没关系”的另一边。
